src/matrixHelpers/FormioScript/scriptGitBridge.js

```diff
@@ -0,0 +1,44 @@
+import { createScripts } from ".";
+
+const version = window.formioQldCdnVersion || "0025/248740";
```

Comment:
gitbridge in squiz is id based after the 0025/, only way to get another 'bridge' is to have another manual gitbridge setup and that would have another random id.

Reply:
so just make the part "248740" variable is fine in this case?